src/core/Microsoft.PowerToys.Settings.UI/Activation/DefaultActivationHandler.cs

```diff
@@ -18,6 +19,7 @@ public DefaultActivationHandler(Type navElement)
             this.navElement = navElement;
         }
 
+        [SuppressMessage("Reliability", "CA2007:Consider calling ConfigureAwait on the awaited task", Justification = "Warning is intended for library code.")]
```

Comment:
As Ryan suggested, we can probably use ConfigureAwait(false) unless there is some context in the thread which would be lost otherwise.